tests/selenium/pages/perfherder.py

```diff
@@ -1,36 +0,0 @@
-from pypom import Region
-from selenium.webdriver.common.by import By
-
-from .base import Base
-
-
-class Perfherder(Base):
```

Comment:
This class definition is still used in this [Treeherder.swith_to_perfherder()](https://github.com/mozilla/treeherder/blob/16263fd09ed3f4c37ca43a35b88ea7fd7a3e1be7/tests/selenium/pages/treeherder.py#L139) method.

I think that's dead code, but it still needs to be addressed. Have we got code coverage for switching between `Treeherder` & `Perfherder` & `Intermittent Failuser View`? If not, we 1st need to provide *and then* delete this class definition.